.idea/.idea.PetroglyphTools/.idea/contentModel.xml

```diff
@@ -1,16 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
All the IntelliJ related stuff needs to be cleaned out and the `.gitignore ` updated properly, but that's for a different issue.

Reply:
See #102